nuxeo-business-days-management-core/src/main/java/org/nuxeo/business/days/management/checker/CSVHolidaysChecker.java

```diff
@@ -65,26 +64,14 @@ private void initDates(String fileName, boolean isEmbedded) {
             return;
         }
 
-        try {
-
-            Reader reader = null;
-
-            if (isEmbedded) {
-                InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream(fileName);
-                reader = new InputStreamReader(is);
-            } else {
-                reader = new FileReader(fileName);
-            }
-
-            BufferedReader buffer = new BufferedReader(reader);
+        try (Reader reader = getReader(fileName, isEmbedded); //
+                BufferedReader buffer = new BufferedReader(reader)) {
```

Comment:
is the Reader needed? `BufferedReader buffer = new BufferedReader(getReader(fileName, isEmbedded)))` ?

Reply:
Yes, this can be simplified.